src/main/java/com/intel/chimera/cipher/CipherFactory.java

```diff
@@ -65,7 +65,7 @@ public static Cipher getInstance(CipherTransformation transformation,
       }
     }
 
-    return (cipher == null) ? new JceCipher(props, transformation) : cipher;
+    return (cipher == null) ? new OpensslCipher(props, transformation) : cipher;
```

Comment:
I would consider this change is dangerous.
Change default cipher to OpensslCipher makes sense but change to OpensslCipher when no Cipher is working may be dangerous. I would suggest, we still use JceCipher here.